src/data/vision/mnist.jl

```diff
@@ -0,0 +1,39 @@
+"""
+    load_mnist()
+
+Loads and prepares MNIST data.
+"""
+function load_mnist(n::Union{Nothing,Int}=nothing)
+    X, y = MLDatasets.MNIST(:train)[:]
+    X = Flux.flatten(X)
+    X = X .* 2.0f0 .- 1.0f0
+    y = MLJBase.categorical(y)
+    counterfactual_data = CounterfactualExplanations.CounterfactualData(
+        X, y; domain=(-1.0, 1.0), standardize=false
+    )
+    counterfactual_data.X = Float32.(counterfactual_data.X)
+    # Undersample:
+    if !isnothing(n)
+        counterfactual_data = CounterfactualExplanations.DataPreprocessing.subsample(
+            counterfactual_data, n
+        )
+    end
+    return counterfactual_data
+end
+
+"""
+    load_mnist_test()
+
+Loads and prepares MNIST test data.
+"""
```

Comment:
Some methods here have short comments, some have long ones - I don't know whether we're planning to tackle the refactoring of comments as part of another PR, but if not, it would be nice to use a standard format for the comments here.

Reply:
this is tackled by #224